crates/ika-core/src/dwallet_mpc/crytographic_computation/mpc_computations/sign.rs

```diff
@@ -158,7 +158,9 @@ impl SignPartyPublicInputGenerator for SignParty {
             }
         };
 
-        let VersionedPresignOutput::V1(presign) = presign;
+        let VersionedPresignOutput::V1(presign) = presign else {
+            todo!("Add support for other versions once Sign supports them")
+        };
```

Comment:
currently the code doesn't work at all, since you are generating v2 presigns but reading v1 only...